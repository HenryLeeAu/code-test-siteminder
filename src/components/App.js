import React, { Component } from 'react';
import SearchBox from 'components/SearchBox';
import MovieList from 'components/MovieList';
import MovieDetail from 'components/MovieDetail';
import Pagination from 'components/Pagination';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div className="sidebar">
          <SearchBox />
          <div className="movieListWrap">
            <MovieList />
          </div>
          <Pagination />
        </div>
        <div className="detail">
          <MovieDetail />
        </div>
      </div>
    );
  }
}

export default App;
