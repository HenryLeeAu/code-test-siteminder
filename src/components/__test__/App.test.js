import React from 'react';

import App from 'components/App'
import { shallow } from 'enzyme'


let  wrappered;
beforeEach(()=>{
    wrappered = shallow(<App />);
});

it('App - render each component', () => {
    expect(wrappered).toMatchSnapshot();
});

