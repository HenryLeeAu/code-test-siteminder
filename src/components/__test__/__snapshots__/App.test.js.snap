// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App - render each component 1`] = `
<div
  className="App"
>
  <div
    className="sidebar"
  >
    <Connect(SearchBox) />
    <div
      className="movieListWrap"
    >
      <Connect(MovieList) />
    </div>
    <Connect(Pagination) />
  </div>
  <div
    className="detail"
  >
    <Connect(MovieDetail) />
  </div>
</div>
`;
